int32 src_uav_id
string component
int32 cmd_priority
string playload
float64 start_time
int32 tgt_uav_count